app/internal/page/fixtures.go

```diff
@@ -67,3 +105,201 @@ func MakeCertificateProvider(firstNames string) actor.CertificateProvider {
 		RelationshipLength:      "gte-2-years",
 	}
 }
+
```

Comment:
I can't comment on the lines, but I've been getting paranoid about emails and mobile numbers, should we set them all to https://docs.notifications.service.gov.uk/rest-api.html#smoke-testing by default so people really can't mess up?

Reply:
Probably the safest bet yeah - I'll add a couple of consts in somewhere that we can reference where we need it in tests.